src/pages/Facility/billing/paymentReconciliation/PaymentReconciliationShow.tsx

```diff
@@ -108,19 +108,12 @@ export function PaymentReconciliationShow({
       {/* Header */}
       <div className="flex flex-col md:flex-row justify-between items-start gap-4">
         <div className="flex items-center gap-2">
-          <Button
-            variant="ghost"
-            className="p-2 h-auto"
-            onClick={() => goBack(`/facility/${facilityId}/billing/payments`)}
-          >
-            <CareIcon icon="l-arrow-left" className="size-5" />
-          </Button>
           <div>
             <h1 className="text-2xl font-bold flex items-center flex-wrap gap-2">
               {t("payment")}
             </h1>
             <span className="text-sm text-gray-500">#{payment.id}</span>
-            <div className="flex gap-2 mt-1 flex-wrap">
+            <div className="grid grid-cols-2 gap-2 mt-1 w-full sm:flex sm:flex-wrap">
```

Comment:
hmm, let's not mix grid and flex, stick to either one.


Design wise, it looks good.